pkg/pmem-csi-driver/controllerserver-node.go

```diff
@@ -215,6 +215,19 @@ func (cs *nodeControllerServer) CreateVolume(ctx context.Context, req *csi.Creat
 
 		cs.mutex.Lock()
 		defer cs.mutex.Unlock()
+		devices, err := cs.dm.ListDevices()
```

Comment:
@okartau I would suggest to modify `DeviceManager.CreateDevice()` to return newly created device. It was my fault, I should have done that while designing the DeviceManager interface.

Reply:
Good point, that makes sense, added a commit doing this